Update readiness-assessment-fix.md (#3936)
MicrosoftDocs/microsoft-365-docs: microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md

### Device Compliance policies

Intune Device Compliance policies in your Azure AD organization may impact Microsoft Managed Desktop devices.

**Not ready**

> You have at least one compliance policy that targets all users. Microsoft Managed Desktop includes compliance policies that will target your MIcrosoft Managed Desktop devices.  Reset the policy to target a specific Azure AD group that does not include any Microsoft Managed Desktop users or devices. For steps, see [Create a compliance policy in Microsoft Intune](https://docs.microsoft.com/mem/intune/protect/create-compliance-policy).

Reviewer: ```suggestion
You have at least one compliance policy that targets all users. Microsoft Managed Desktop includes compliance policies that will target your Microsoft Managed Desktop devices.  Reset the policy to target a specific Azure AD group that does not include any Microsoft Managed Desktop users or devices. For steps, see [Create a compliance policy in Microsoft Intune](https://docs.microsoft.com/mem/intune/protect/create-compliance-policy).
```

**Not ready**

> You currently have at least one enrollment restrictions policy configured to prevent Windows devices from enrollment into Intune. Follow the steps in [Set enrollment restrictions](https://docs.microsoft.com/mem/intune/enrollment/enrollment-restrictions-set) for each enrollment restriction policy that targets Microsoft Managed Desktop users and change the Windows (MDM) setting to **Allow**.  Note that personally owned WIndows (MDM) devices can be set to **Block**. 

Reviewer: ```suggestion
You currently have at least one enrollment restrictions policy configured to prevent Windows devices from enrollment into Intune. Follow the steps in [Set enrollment restrictions](https://docs.microsoft.com/mem/intune/enrollment/enrollment-restrictions-set) for each enrollment restriction policy that targets Microsoft Managed Desktop users and change the Windows (MDM) setting to **Allow**.  Note that personally owned Windows (MDM) devices can be set to **Block**. 
```

### Intune enrollment

Windows 10 devices in your Azure AD organization must be able to automatically enroll in Intune.

**Advisory**

Make sure the MDM User scope is set to **Some** or **All**, not **None**. If you choose **Some**, come back after enrollment and select the **Modern Workplace -All** Azure AD group for **Groups** or an equivalent group targeting all of your Microsoft Managed Desktop users.

### Microsoft Store for Business

> We use Microsoft Store for Business and deploy the Company Portal app on Microsoft Managed Desktop so allow users to optionally install some apps, such as Microsoft Project and Microsoft Visio (where pemritted).

Reviewer: ```suggestion
We use Microsoft Store for Business and deploy the Company Portal app on Microsoft Managed Desktop to allow users to optionally install some apps, such as Microsoft Project and Microsoft Visio (where permitted).
```

**Not ready**

> Microsoft Store for Business either isn't enabled or isn't synced with Intune. For more information, see [How to manage volume purchased apps from the Microsoft Store for Business with Microsoft Intune](https://docs.microsoft.com/mem/intune/apps/windows-store-for-business) and [Install Intune Company Portal on on devices](../get-started/company-portal.md).

Reviewer: ```suggestion
Microsoft Store for Business either isn't enabled or isn't synced with Intune. For more information, see [How to manage volume purchased apps from the Microsoft Store for Business with Microsoft Intune](https://docs.microsoft.com/mem/intune/apps/windows-store-for-business) and [Install Intune Company Portal on devices](../get-started/company-portal.md).
```
